### PR TITLE
fix(grid): default select displays custom perpage as intended #4207

### LINKED ISSUE #4207: [Grid] Default pagination template with a custom value for the per page property
## Description  
Custom `perPage` values are not correctly reflected in the default pagination template.

 * igniteui-angular version: 7.1+
 * browser: All

## Steps to reproduce  

1. Open [this sample](https://stackblitz.com/edit/angular-hxnrvv)

## Result  
The grid per page property is set to 12 and the grid has 12 rendered rows. However the default select shows 5.

## Expected result  
The select in the default pagination template should reflect the value.

### PR DESCRIPTION
**Pull request by VladB93**
Closes #4207 

### Additional information (check all that apply):
 - [X] Bug fix
 - [ ] New functionality
 - [ ] Documentation
 - [ ] Demos
 - [ ] CI/CD

### Checklist:
 - [X] All relevant tags have been applied to this PR
 - [X] This PR includes unit tests covering all the new code
 - [ ] This PR includes API docs for newly added methods/properties
 - [ ] This PR includes `feature/README.MD` updates for the feature docs
 - [ ] This PR includes general feature table updates in the root `README.MD`
 - [ ] This PR includes `CHANGELOG.MD` updates for newly added functionality
 - [ ] This PR contains breaking changes
 - [ ] This PR includes `ng update` migrations for the breaking changes
 - [ ] This PR includes behavioral changes and the feature specification has been updated with them

### CODE REVIEW

File: projects/igniteui-angular/src/lib/grids/grid/grid.component.spec.ts [hunk 1]
```diff
@@ -3711,6 +3711,33 @@ describe('IgxGrid Component Tests', () => {
                     expect(overlayContent.style.display).toEqual('');
                 }));
         });
+
+        describe('Selector should display random numbers from perpage', () => {
```
Comment (nrobakova) [inline projects/igniteui-angular/src/lib/grids/grid/grid.component.spec.ts]: This test should be added in the grid.pagination.spec.ts. as TestBed can be used for example PagingComponent. 
There is no need test to be async. 
Tests for changing grid.perPage to default values should be added.

File: projects/igniteui-angular/src/lib/grids/grid-common.pipes.ts [hunk 2]
```diff
@@ -87,3 +87,13 @@ export class IgxGridTransactionPipe implements PipeTransform {
         return collection;
     }
 }
+
+@Pipe({
```
Comment (bkulov) [inline projects/igniteui-angular/src/lib/grids/grid-common.pipes.ts]: Add documentation or `/** @hidden */`.

File: projects/igniteui-angular/src/lib/grids/grid-common.pipes.ts [hunk 3]
```diff
@@ -87,3 +87,13 @@ export class IgxGridTransactionPipe implements PipeTransform {
...
+    name: 'filterUniqueOptions',
+    pure: true,
+})
+export class IgxGridUniqueOptionsPipe implements PipeTransform {
```
Comment (bkulov) [inline projects/igniteui-angular/src/lib/grids/grid-common.pipes.ts]: Please choose more meaningful name.

File: projects/igniteui-angular/src/lib/core/i18n/grid-resources.ts [hunk 4]
```diff
@@ -149,5 +150,6 @@ export const GridResourceStringsEN: IGridResourceStrings = {
     igx_grid_excel_number_filter: 'Number filter',
     igx_grid_excel_date_filter: 'Date filter',
     igx_grid_excel_boolean_filter: 'Boolean filter',
-    igx_grid_excel_custom_filter: 'Custom filter...'
+    igx_grid_excel_custom_filter: 'Custom filter...',
+    igx_grid_paginator_label: 'Items per page'
```
Comment (kdinev) [inline projects/igniteui-angular/src/lib/core/i18n/grid-resources.ts]: Can we simplify this to "per page".

**Pull request merged**